pages/about.html

```diff
@@ -1,12 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="css/style.css" rel="stylesheet">
+    <link href="css/about.css" rel="stylesheet">
+    <title>About Page</title>
+  </head>
+  <body>
+    <header>
+      <nav class="topnav">
+        <a href="file:///Users/nina/Developer/projects/personal-portfolio-site/pages/index.html">Home</a>
+        <a href="file:///Users/nina/Developer/projects/personal-portfolio-site/pages/portfolio.html">Portfolio</a>
```

Comment:
Careful with your file paths! This path points to a specific file on your specific computer. Other people do not have access to that file. As soon as this goes live in the Internet, the links will not work. Instead of using an absolute path, we can use a relative path instead. In this case, it would be `index.html` and `portfolio.html` because those two files are in the same directory as the `about.html` file. The links for your pictures are also broken and do not show up. For the images, the link would be `../images/nina-pic.jpeg`.

Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/